centreon/src/Core/AgentConfiguration/Application/Validation/CmaValidator.php

```diff
@@ -79,6 +79,9 @@ private function validateAgentInitiatedConnection(array $configuration, Connecti
         if ($configuration['agent_initiated'] === false) {
             return;
         }
+        if ($configuration['port'] === null) {
+            throw AgentConfigurationException::portIsMandatoryForAgentInitiated();
```

Comment:
I think this message is confusing as port is now mandatory in BOTH agent initiated and poller initiated connection
I would remove the 'when ....' part